arcgis-ios-sdk-samples/Layers/Edit KML ground overlay/EditKMLGroundOverlay.storyboard

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="17156" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="NFA-Uf-2BM">
+    <device id="retina6_1" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="17125"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="System colors in document resources" minToolsVersion="11.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--EditKML Ground Overlay View Controller-->
+        <scene sceneID="5c7-Yi-wLK">
+            <objects>
+                <viewController id="NFA-Uf-2BM" customClass="EditKMLGroundOverlayViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="VhX-3L-oDy">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="QlX-Cr-Hln" userLabel="Scene View" customClass="AGSSceneView">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="812"/>
+                                <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                            </view>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="OBw-N8-s44" userLabel="Slider View">
```

Comment:
Did you try using a toolbar for this view? It is positioned where a toolbar should be, but it doesn't have the right background color or the right size. Also, the spacing on the right of the view is different than the spacing on the left. Rather than using a constant for the spacing, the first and last views should be constrained to the _margins_ of the superview:

![image](https://user-images.githubusercontent.com/2257493/94060048-2a044280-fd98-11ea-98f9-bc2c7d6db0fe.png)

That will use the default margins, but they can be changed if desired:

![image](https://user-images.githubusercontent.com/2257493/94060307-9b43f580-fd98-11ea-8c17-2f1ccc18766f.png)

Reply:
I considered using a toolbar but decided against it because the only "button" is the slider. I also saw a different sample that put the view inside a toolbar. Would it be best to just put the slider in a toolbar without a view?

Reply:
It is OK to use a toolbar, since our only control is a slider. And we've done that in samples such as #922 .

However, putting a slider in toolbar as a bar button item fixes its width. It may look bad on iPad landscape with a tiny slider in the middle.
It is also not easy to configure the labels in a toolbar (need to embed them in views and set constraints). So I think either using toolbar, or use a custom wrapper view or stack view is fine.